enable/tools/hover_tool.py

```diff
@@ -7,9 +7,9 @@
 
 # Enthought library imports
 from enable.base_tool import BaseTool
-from traits.etsconfig.api import ETSConfig
-from pyface.toolkit import toolkit_object
 from traits.api import Any, Callable, Enum, Float, Int
+from traits.etsconfig.api import ETSConfig
+from pyface.timer.api import Timer
```

Comment:
And this would become:
```suggestion
from pyface.timer.api import do_after
```